### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by Paahn**
- Amendment to README to inform new developers verifiable credentials are not needed to run the project

### CODE REVIEW

File: documentation/VerifiableCredentials.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ## Running the Aries Agent Locally for Verifiable Credential Testing and Development
 
+**Please Note** The following process is only necessary when working with Verifiable Credential Testing and Development, ie not required when initially setting up the project.
```
Comment (mtpultz) [inline documentation/VerifiableCredentials.md:3]: Looks great!!!  

I would probably use `**NOTE:**`, and replace `ie` with `, and is not necessary for general development purposes.  For example, when initially setting up the project.`

**Review by mtpultz — APPROVED** [review]
Woot! Someone in the future will thank you!

File: documentation/VerifiableCredentials.md (hunk 1 above)
Comment (neophyte57) [reply documentation/VerifiableCredentials.md]: Can we eliminate the sentence fragment, i.e. `... general development purposes, for example, when ...`

**Review by AnaisHebert — APPROVED** [review]
🦆🦆🦆

File: documentation/VerifiableCredentials.md (hunk 1 above)
Comment (mtpultz) [reply documentation/VerifiableCredentials.md]: If you want to eliminate the sentence fragment it would be `purposes; for example, when initially`

**Pull request merged**